middle_end/base_types/closure_origin.ml

```diff
@@ -0,0 +1,22 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                       Pierre Chambart, OCamlPro                        *)
+(*     Mark Shinwell, Leo White and Fu Yong Quah, Jane Street Europe      *)
+(*                                                                        *)
+(*   Copyright 2013--2017 OCamlPro SAS                                    *)
+(*   Copyright 2014--2017 Jane Street Group LLC                           *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+[@@@ocaml.warning "+a-4-9-30-40-41-42"]
+
+include Closure_id
+
+let create t = t
```

Comment:
As this isn't creating a fresh identifier, this should rather be called 'wrap'

Reply:
I'm not sure I agree about this.  From the outside, this is effectively creating a fresh identifier, even if under the hood it is re-using an existing one.